config/production.yaml

```
@@ -23,7 +23,7 @@ jira: {
 }
 features:
   submitEndpointForm:
-    enabled: false
+    enabled: true
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)